src/content/tests.py

```diff
@@ -1,3 +1,30 @@
-from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
 
-# Create your tests here.
+
+class ContentViewTest(TestCase):
+    def setUp(self):
+        self.c = Client()
+        self.c.force_login(User.objects.get_or_create(username="testuser")[0])
+
+    def test_index_should_display_news(self):
+        from .views import index
+        from triage.models import News, Rating
+
+        rating = Rating(name="rating1")
+        rating.save()
```

Comment:
To create and save an object in a single step, you can use the [create() method](https://docs.djangoproject.com/en/3.0/topics/db/queries/#creating-objects): `Rating.objects.create(...)`